src/popup/router/index.ts

```diff
@@ -69,7 +69,9 @@ router.beforeEach(async (to, from, next) => {
     }[POPUP_TYPE];
 
     if (name !== to.name) {
-      next({ name, params: await getPopupProps() as Dictionary });
```

Comment:
After that the saved `popupProps` would not be removed from the state, and will be `v-bind` to every mounted component which is bad.
Previous approach stoped working becuase of that https://github.com/vuejs/router/blob/main/packages/router/CHANGELOG.md#414-2022-08-22
We have several options:
1) We can use one of the approaches the `vue-router` team suggesting.
2) We can write a composable that will load this info and use it for every modal that can be accesed in popup.
3) We can use `JSON.stringify` to `app` object and then `JSON.parse` it back in the popup.

Also it means that every place in the project where prop is not defined in the query with new vue-router it is not working anymore.